service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <service verb="get" noun="PackageMap">
+        <in-parameters>
+            <parameter name="shipmentPackageSeqId" required="true"/>
+            <parameter name="boxId" required="true"/>
+            <parameter name="weightUomId" required="true"/>
```

Comment:
Should this be required field?

Reply:
@gurveenbagga Yes mam, unit of weight is the required field in the requestMap.